### ISSUE TITLE
win_updates expansion 

### ISSUE THREAD

**Issue by dagwieers** [ansible/ansible#25282]
_From @michelleperz on September 4, 2015 17:15_

##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

win_updates module
##### ANSIBLE VERSION

N/A
##### SUMMARY

To mitigate the risk of pushing out security updates to prod that may not have been tested on dev or staging, it would be nice to have an additional option in this module wherein updates could be applied based on KB#. 

_Copied from original issue: ansible/ansible-modules-extras#921_

**Comment by dagwieers** [comment]
_From @trondhindenes on October 6, 2015 22:29_

That's a great idea. The underlying module (PsWindowsUpdate) supports installing a specific KB thru the "-UpdateID" param. If we surfaced that as an Ansible param/option would that solve your problem?

**Comment by dagwieers** [comment]
_From @ansibot on August 4, 2016 22:28_

@mattdavispdx, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @ansibot on September 13, 2016 3:38_

@mattdavispdx, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by dagwieers** [comment]
_From @jhawkesworth on September 14, 2016 11:16_

hmm, I imagine bot should be poking @nitzmahone  (perhaps just misconfigured).

bot_broken

**Comment by dagwieers** [comment]
_From @ansibot on December 7, 2016 14:8_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by dagwieers** [comment]
@michelleperz Could we please move this issue over to https://github.com/ansible/ansible if this is still considered valuable ? Otherwise it may not get the proper attention it may need.

Thanks in advance !

**Comment by gmreburn** [comment]
This feature would be useful for me. I like @trondhindenes's idea to pass the UpdateID to PsWindowsUpdate.

**Comment by jctanner** [comment]
!bot_broken

**Comment by dagwieers** [comment]
@jborean93 Isn't this already implemented ?

**Comment by jborean93** [comment]
Yep I mus have missed this issue, will be available in the 2.5 release.
